src/server/index.tsx

```diff
@@ -58,19 +58,24 @@ server.get("/robots.txt", async (_req, res) => {
   res.send(robotstxt);
 });
 
-server.get("/css/themes/:name", async (req, res) => {
+server.get("/css/themes/:set?/:name", async (req, res) => {
   res.contentType("text/css");
   const theme = req.params.name;
   if (!theme.endsWith(".css")) {
     res.send("Theme must be a css file");
   }
 
-  const customTheme = path.resolve(`./${extraThemesFolder}/${theme}`);
-  if (fs.existsSync(customTheme)) {
-    res.sendFile(customTheme);
+  if (req.params.set === 'hljs') {
+    const hljsTheme = path.resolve(`./node_modules/highlight.js/styles/${theme}`);
```

Comment:
IDK how you feel about pulling files from the node_modules folder, but we could webpack CopyPlugin the files to `src/assets/css/themes/hljs` or something if you prefer.

Reply:
Yep I like that better. hljs-light.css and hljs-dark.css . Also a symlink might work but I'm not sure.